ci/request-reviews/request-reviews.sh

```diff
@@ -80,7 +80,7 @@ if ! "$SCRIPT_DIR"/verify-base-branch.sh "$tmp/nixpkgs.git" "$headRef" "$baseRep
 fi
 
 log "Getting code owners to request reviews from"
-"$SCRIPT_DIR"/get-reviewers.sh "$tmp/nixpkgs.git" "$baseBranch" "$headRef" "$ownersFile" "$prAuthor" > "$tmp/reviewers.json"
+"$SCRIPT_DIR"/get-reviewers.sh "$tmp/nixpkgs.git" "$baseBranch" "$headRef" "$ownersFile" "$prAuthor" "$baseRepo" "$prNumber" > "$tmp/reviewers.json"
```

Comment:
Comment warning: complete nitpicking.

As a matter of order, what makes the most sense to me is:

1. headRef
2. ownersFile
3. baseRepo
4. baseBranch
5. prNumber
6. prAuthor

plus then aligning these names with the names of the variables in the get-reviewers script.

Reply:
Done, but I moved `headRef` before `prNumber` because both are related to the PR :)